components/footer.jsx

```diff
@@ -98,16 +100,14 @@ var Footer = React.createClass({
               <ul className="list-unstyled">
                 <li><a href="https://webmaker.org/">Webmaker</a></li>
                 <li><a href="https://sendto.mozilla.org/">Donate</a></li>
-                <li><a href="">Legal</a></li>
-                <li><a href="">Privacy</a></li>
+                <li><a href="">Twitter</a></li>
               </ul>
             </div>
             <div className="col-xs-6">
               <ul className="list-unstyled">
-                <li><a href="">Contact</a></li>
-                <li><a href="">Twitter</a></li>
-                <li><a href="">Facebook</a></li>
-                <li><a href="">Partners</a></li>
+                <li><a href="">Legal</a></li>
```

Comment:
I'm actually waiting to hear from moco legal about that. pinged again yesterday. @vasquez did design those pages, so maybe we should just build them. i'll file a ticket to do that.